app/src/main/java/com/example/tasky/feature_agenda/domain/model/Event.kt

```diff
@@ -0,0 +1,14 @@
+package com.example.tasky.feature_agenda.domain.model
+
+data class Event(
+    val id: String,
+    val title: String,
+    val description: String,
+    val from: Long,
+    val to: Long,
+    val remindAt: Long,
+    val host: String,
+    val isUserEventCreator: Boolean,
+    val attendees: List<Attendees>,
+    val photos: List<Photo>
+)
```

Comment:
This would then be your DTO model (and should be renamed to `EventDto`) and then you'll create an additional `Event` model in domain which contains just the information you need across the project. Also, as I said, since you're mapping this, you can use different field types whenever it's more convenient